tests/opentracer/test_tracer_gevent.py

```diff
@@ -42,7 +42,7 @@ def g():
                 gevent.sleep(0.03)
                 span.set_tag("g", "yes")
 
-        with ot_tracer.start_span("root"):
+        with ot_tracer.start_active_span("root"):
```

Comment:
what's the difference here?

Reply:
The test case actually assumed that the the span was activated. It was a bug in the previous implementation that we would always activate spans (`start_span` in opentracing doesn't activate spans by default) and so the test passed.